src/app/website/company-home/company-home-routing.module.ts

```diff
@@ -0,0 +1,19 @@
+import { CompanyHomeComponent } from './company-home.component';
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+
+const compARoutes: Routes = [
```

Comment:
Comp`A`Routes?

Reply:
actually I tried it in different sample code and used here I will change this